app/controllers/clearance/passwords_controller.rb

```diff
@@ -32,7 +32,7 @@ def edit
 
     if params[:token]
       session[:password_reset_token] = params[:token]
-      redirect_to edit_user_password_url(@user)
+      redirect_to url_for
```

Comment:
I've not seen `url_for` used with no arguments before and I can't tell from first glance at the documentation what it's supposed to do. It it equivalent to `url_for :edit`, where `:edit` is the current action?